Magic/UI/TypeCell.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  TypeCell.swift
+//  Magic
+//
+//  Created by filipe.n.jordao on 07/05/19.
+//
+
+import UIKit
+
+final class TypeCell: UICollectionViewCell {
+    let typeLabel = UILabel()
```

Comment:
Humm, suggestion:

`
let typeLabel: UILabel = {
     let label = UILabel(frame: .zero)
      label.textAlignment = .left
	  label.textColor = #colorLiteral(red: 0.8318908215, green: 0.8277270198, blue: 0.8056753278, alpha: 1)
	  label.font = UIFont.boldSystemFont(ofSize: 14)
     return label
 }()
`

Using this kind of approach each component configure yourself at initialisation, since there are static values and you don't need to call `setupViews`

Reply:
Okay